static/css/business.less

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+   cfgov-refresh
+   Business with us pages at /doing-business-with-us/.
+   ========================================================================== */
+
+// Used to set padding to 30px
+@padding: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
This could maybe be pulled out into a higher-level variable. I think 30px and 15px gaps in ems are a common size we use and not just on this page.